pyfolio/tears.py

```diff
@@ -512,16 +501,15 @@ def create_returns_tear_sheet(returns, positions=None,
 
     plotting.show_worst_drawdown_periods(returns)
 
-    # If the strategy's history is longer than the benchmark's, limit strategy
-    if returns.index[0] < benchmark_rets.index[0]:
-        returns = returns[returns.index > benchmark_rets.index[0]]
-
-    vertical_sections = 13
+    vertical_sections = 11
```

Comment:
Confirming for my own understanding: we removed a section here, right? We used to default to 13. Now we default to 11 and increment by 1 if benchmark_rets was passed. And the removed section corresponds to the deletion of `ax_rolling_risk = plt.subplot(gs[i, :], sharex=ax_rolling_returns)` below?

Reply:
That's correct!